docs/UserGuide.md

```diff
@@ -82,32 +82,31 @@ Shows a message explaning how to access the help page.
 
 Format: `help`
 
+### Listing all contacts : `contacts`
+
+Shows a list of all contacts in address book.
+
+Format: `contacts`
 
-### Adding a person: `add`
+### Adding a person: `addcontact`
 
 Adds a person to the address book.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+Format: `addcontact n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 A person can have any number of tags (including 0)
 </div>
 
 Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `addcontact n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
+* `addcontact n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
 
-### Listing all contacts : `contacts`
-
-Shows a list of all contacts in address book.
-
-Format: `contacts`
-
-### Editing a person : `edit`
+### Editing a person : `editcontact`
 
 Edits an existing person in the address book.
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
+Format: `editcontact INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
```

Comment:
Edit contact has brackets but addcontact does not.
Need to standardise.

Reply:
Thats because `editcontact` tags are optional but `addcontact` are not. Have a look at the original ab3